timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/PaxosTimeLockServerIntegrationTest.java

```diff
@@ -212,7 +209,8 @@ private int lockAndUnlockAndCountExceptions(List<LockService> lockServices, int
         AtomicInteger exceptionCounter = new AtomicInteger(0);
         futures.forEach(future -> {
             try {
-                assertNull(future.get());
+                LockRefreshToken test = future.get();
+                assertNull(test);
```

Comment:
`assertThat(test).isNull()`

there are like 2 instances each of `assertNull` `assertTrue` and `assertNotNull`, assertj-ize it